api/src/main/java/de/adorsys/keymanagement/api/keystore/KeyStoreView.java

```diff
@@ -35,7 +35,23 @@
      */
     <Q> AliasView<Q> allAliases();
 
+    /**
+     * Get source of keys
+     * @return object which have access to keys and can operate with them
+     */
     KeySource source();
+
+    /**
+     * Generates keySet from all existing in key source keys
+     * @param keyPassword password which is used to access keys
+     * @return keySet with lists of key pairs, secret keys or key entries
+     */
     KeySet copyToKeySet(Function<String, char[]> keyPassword);
+
+    /**
+     * Generates template from all existing in key source keys
```

Comment:
`Clones current KeySource into KeySetTemplate in a way, that all keys will be materialized in KeySetTemplate. You can view it as generating template from all existing in key source keys`

Reply:
done